tests/test_parse_KEGG.py

```diff
@@ -0,0 +1,223 @@
+
+
+def test_parse_gl(gl_raw_record):
+    co_record = parse_co(gl_raw_record)
+    assert len(co_record) == 6
+    assert len(co_record['REACTION']) == 2
+    assert co_record['PATHWAY'][0] == ('map00000', 'Fake pathway')
+
+
+@pytest.fixture()
+def pathway_raw_record():
+    return "ENTRY       ko00000                      Pathway\n" \
+           "NAME        a fake pathway\n" \
+           "DESCRIPTION A long description of what this pathway does\n" \
+           "CLASS       09100 Metabolism; 09101 Carbohydrate metabolism\n" \
+           "PATHWAY_MAP ko00000 a fake pathway\n" \
+           "COMPOUND    C00000 A fake compound\n" \
+           "            C99999 Another fake compound\n" \
+           "DBLINKS     GO: 0019852 0019572\n" \
+           "ORGANISM    Homo sapiens (human) [GN:hsa]\n" \
+           "REFERENCE\n" \
+           "  AUTHORS   Michal G.\n" \
+           "  TITLE     Biochemical Pathways\n" \
+           "  JOURNAL   Wiley (1999)\n"
+
+
+def test_parse_pathway(pathway_raw_record):
+    pathway_record = parse_pathway(pathway_raw_record)
+    assert len(pathway_record) == 7
+    assert set([compound[0] for compound in pathway_record['COMPOUND']]) == {'C00000', 'C99999'}
+
+
+@pytest.fixture()
+def organism_raw_record():
+    return "ENTRY       000000            CDS       H.sapiens\n" \
+           "NAME        SBS6969, ASBS1234\n" \
+           "DEFINITION  a fake guman gene\n" \
+           "ORTHOLOGY   K00000 Some fake orthology\n" \
+           "PATHWAY     hsa00000  Specific Stuff\n" \
+           "            hsa99999  The Place\n" \
+           "CLASS       Things; Stuff; Specific Stuff [PATH:hsa00000]\n" \
+           "            This; Must Be; The Place [PATH:hsa99999]\n" \
+           "POSITION    1p1.1\n" \
+           "MOTIF       Pfam: ASDF, JKL1, HYDE\n" \
+           "            PROSITE: A_PROSITE_ID ANOTHER_PROSITE_ID\n" \
+           "DBLINKS     NCBI-GI: 00000000\n" \
+           "            HGNC: 00000\n" \
+           "AASEQ       12\n" \
+           "            QRHMIRHTGDGPYKCQECGKAFDRPSLFRIHERTHTGEKPHECKQCGKAFISFTNFQSHM\n" \
+           "            IRHTGDGPYKCKVCGRAFIFPSYVRKHERTHTGEKPYECNKCGKTFSSSSNVRTHERTHT\n" \
+           "            GEKPYECKECGKAFISLPSVRRHMIKHTGDGPYKCQVCGRAFDCPSSFQIHERTHTGEKP\n" \
+           "NTSEQ       1234\n" \
+           "            agaactcacactggtgagaaaccctatgcatgtccggaatgtgggaaagccttcatttct\n" \
+           "            ctcccaagtgttcgaagacacatgattaagcacactggagatggaccatataaatgtcag\n" \
+           "            gaatgtgggaaagcctttgatcgcccaagtttatttcagatacatgaaagaactcacact\n" \
+           "            ggagagaaaccctatgaatgtcaggaatgtgcaaaagctttcatttctcttccaagtttt\n"
+
+
+def test_parse_organism(organism_raw_record):
+    organism_record = parse_organism(organism_raw_record)
+    assert len(organism_record) == 9
+    assert len(organism_record['ORTHOLOGY']) == 2
+    assert organism_record['ORTHOLOGY'][0] == 'K00000'
+
+
+@pytest.fixture()
```

Comment:
Move all below to new tests/test_downloader.py file with tests for downloader.py.

Reply:
I accidentally committed them to the other branch